web/client/actions/catalog.js

```diff
@@ -133,6 +134,22 @@ function recordsLoadError(e) {
         error: e
     };
 }
+
+function catalogInited() {
+    return {
+        type: CATALOG_INITED
+    };
+}
+
+function initCatalog(apis = API) {
+    return (dispatch) => {
+        Object.keys(apis).forEach((name) => {
+            apis[name].reset();
+        });
+        dispatch(catalogInited());
```

Comment:
Waiting for re-implementation using epics, using a thunk is ok... But I didn't get the utility of this action call here.